test/studies/shootout/mandelbrot/ldelaney/mandelbrot-release-dynamic.chpl

```diff
@@ -0,0 +1,62 @@
+/* The Computer Language Benchmarks Game
+   http://benchmarksgame.alioth.debian.org/
+
+   contributed by Jacob Nelson, Lydia Duncan, Brad Chamberlain,
+   and Ben Harshbarger
+   derived from the GNU C version by Greg Buchholz
+*/
+
+use DynamicIters;
+
+config const n = 200,              // image size in pixels (n x n)
+             maxIter = 50,         // max # of iterations per pixel
+             limit = 4.0,          // per-pixel convergence limit
+             chunkSize = 1;        // dynamic iterator's chunk size
+
+param bitsPerElt = 8;              // # of bits to store per array element
+type eltType = uint(bitsPerElt);   // element type used to store the image
+
+
+proc main() {
+  const xdim = 0..#divceilpos(n, bitsPerElt);  // the compacted x dimension
+
+  var image : [0..#n, xdim] eltType;            // the compacted bitmap image
```

Comment:
@Ldelaney and @benharsh : When moving this to the release, what do you think about simply capturing the expression divceilpos(n, bitsPerElt) in a const named something like 'nPacked', eliminating xdim, and using an anonymous range for the x dimension as well?  I.e.,

    const nPacked = divceilpos(n, bitsPerElt);

    var image: [0..#n, 0..#nPacked] eltType;

This has the advantage of not having an xdim without a ydim, avoids the ongoing strangeness of why the second dimension is 'x' (though I guess the loop below preserves it in any case), and makes the dimensions of image seem slightly more symmetrical in how they're declared.